lib/gssapi/spnego/init_sec_context.c

```diff
@@ -415,21 +415,13 @@ spnego_reply
     if (ret)
       return ret;
 
+    /* The SPNEGO token must be a negTokenResp */
     if (resp.element != choice_NegotiationToken_negTokenResp) {
 	free_NegotiationToken(&resp);
 	*minor_status = 0;
 	return GSS_S_BAD_MECH;
     }
 
-    if (resp.u.negTokenResp.negResult == NULL
```

Comment:
We still need to handle `resp.u.negTokenResp.negResult == NULL` somewhere

Reply:
Yes.  What's it mean?  :)

Reply:
`GSS_S_DEFECTIVE_TOKEN`

Reply:
reject :)

Reply:
I presume... BTW is that misnamed or from an earlier version? RFC 4178 says it's called negState. It's also only required on the first reply from the acceptor. So, hmm, maybe I need to look at this more carefully – if it is absent we should still process a context token that may be an error token.

Reply:
the local negState variable may be overkill, just looked cleaner to me. Presumably the ctx->open and else branches can't be reached, unless you can restart negotiation (which I don't think you can do with SPNEGO?)